css/style.css

```diff
@@ -0,0 +1,111 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%;
+}
+
+@media only screen and (max-width: 600px) {
+  body {
+    font-family: "Nunito", sans-serif;
+    color: #28352f;
+    font-weight: 300;
+    line-height: 1.6;
+  }
+  .Main-container {
+    background-image: url(../img/header-illsutration-mobile.png);
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+    background-size: 100% 40%;
+    height: 1400px;
```

Comment:
[OPTIONAL] I see that the previous reviewer requested you remove the fixed width. I recommend you give a `{width of 100% }`to this container. This is to support all device widths. Currently your app is only supporting galaxy ii
<img width="849" alt="Screenshot 2022-09-23 at 12 10 47" src="https://user-images.githubusercontent.com/48546992/191930496-669b20c5-8f4d-4e49-8cce-84f433af0eea.png">

- [ ] This is how it appears on a wider device, 600px
<img width="704" alt="Screenshot 2022-09-23 at 12 10 56" src="https://user-images.githubusercontent.com/48546992/191930618-14c92e40-2131-4f20-8e6e-758beec6ae7a.png">